consensus/src/consensus/consensus_proxy.rs

```diff
@@ -125,6 +126,7 @@ impl<N: Network> ConsensusProxy<N> {
                     RequestTransactionReceiptsByAddress {
                         address: address.clone(),
                         max,
+                        start_at: start_at.clone(),
```

Comment:
This `.clone()` is likely unnecessary.
```suggestion
                        start_at,
```

Reply:
No, because it's in a loop